<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTrnLedger</class>
 <widget class="QMainWindow" name="frmTrnLedger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LEDGER</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbTag_5">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>52</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="title">
     <string>OPTION</string>
    </property>
    <widget class="QComboBox" name="cboLgrCus">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>29</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_59">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>27</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>CUSTOMER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_60">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>MACHINE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cboLgrMach">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>52</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFetch">
     <property name="geometry">
      <rect>
       <x>469</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>9</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Century Gothic&quot;;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>FETCH</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>5</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/icons/957216-ui-interface/png/094-agenda.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>13</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 24pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>LEDGER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbCustList_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>571</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="title">
     <string>Record(s) Found</string>
    </property>
    <widget class="QTableWidget" name="tblLedger">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>551</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>DATE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TRN #</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PARTICULARS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CREDIT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DEBIT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BALANCE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STATUS</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnFetch</sender>
   <signal>clicked()</signal>
   <receiver>frmTrnLedger</receiver>
   <slot>fillLedgerRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboLgrCus</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>frmTrnLedger</receiver>
   <slot>fillComboDataPassVal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>managePaymentSlot()</slot>
  <slot>fillComboDataPassVal()</slot>
  <slot>fetchAcctRecord()</slot>
  <slot>fillLedgerRecord()</slot>
 </slots>
</ui>
